maven/lib/dependabot/maven/file_fetcher.rb

```diff
@@ -25,13 +25,18 @@ def fetch_files
         fetched_files << pom
         fetched_files += child_poms
         fetched_files += relative_path_parents(fetched_files)
+        fetched_files << extensions if extensions
         fetched_files.uniq
       end
 
       def pom
         @pom ||= fetch_file_from_host("pom.xml")
       end
 
+      def extensions
+        @extensions ||= fetch_file_if_present(".mvn/extensions.xml")
```

Comment:
Spotted this fails when the repo doesn't include a `.mvn` directory
```suggestion
        return @extensions if defined?(@extensions)
        
        @extensions =
          begin
            fetch_file_if_present(".mvn/extensions.xml")
          rescue Dependabot::DependencyFileNotFound
            nil
          end

```